app/commands/decidim/vocdoni/admin/update_election_calendar.rb

```diff
@@ -51,7 +51,7 @@ def election_type_attributes
               auto_start: !form.manual_start,
               secret_until_the_end: form.result_type == "after_voting",
               interruptible: Decidim::Vocdoni.interruptible_elections,
-              dynamic_census: form.dynamic_census,
+              dynamic_census: election.internal_census? ? true : form.dynamic_census,
```

Comment:
I think the `dynamic_census` attribute is not used at all in the form right?
maybe we can just use
```suggestion
              dynamic_census: election.internal_census?
```

Reply:
An then remove the property `dynamic_census` entirerly from the form